app/controllers/teams_controller.rb

```diff
@@ -64,15 +64,19 @@ def team
   end
 
   def team_params
-    params.require(:team).permit(:name, :agency_id, :description, user_ids: [])
+    params.require(:team).permit(:name, :agency_id, :description)
+  end
+
+  def user_params
+    params[:user_ids]&.split(' ') || []
   end
 
   def update_params_with_current_user
     if current_user.admin?
       team_params
     else
-      user_ids = team_params[:user_ids] || []
-      team_params.merge(user_ids: (user_ids << current_user.id.to_s))
+      user_ids = user_params
+      team_params.merge(user_ids: (user_ids << current_user.id.to_s).uniq)
```

Comment:
Is this used for removing team members or just for adding?

Reply:
just adding

Reply:
well that is a little misleading. they are added in a separate form, unless the team is being created. if it's being created, then line 79 is how the current_user is added to the team. if the team is being updated, the team members actually already exist, but since it's using the same code path as create, if they are not explicitly passed through, then the user_ids are JUST the current_user, and the existing users get removed as part of the update. 

this is one solution -- as my comment above notes, there is another one, which is altering what the create path uses. (i don't really love either option and am extremely open to feedback about what is preferred!)

Reply:
I feel like it might be a little less error-prone if we had a separate array for the existing users (the one in the template) and also did a non-mutating `+` instead of `<<` but maybe a candidate for a followup PR

so the end result could look like
```
team_params.merge(user_ids: (existing_user_ids + [current_user_id.to_s]).uniq)
```

Reply:
happy to make that change in this pr!